README.md

```diff
@@ -1,2 +1,320 @@
-# roca
-A command-line tool for running brightscript tests
+# Roca
+
+BrightScript unit testing.  No device required.
+
+## Getting Started
+Note: All examples use `npm`, but these steps should be easily adapted to [yarn](https://yarnpkg.com/), [pnpm](https://pnpm.js.org/), and many other JavaScript package managers.
+
+### Install
+Roca requires the [brs](https://github.com/sjbarag/brs/) runtime, which runs on [node.js](https://nodejs.org/en/).  If needed, initialize an NPM project with:
+
+```bash
+npm init
+```
+
+Then install `@hulu/roca` and `brs` as development dependencies with:
+
+```bash
+npm install --save-dev @hulu/roca brs
+```
+
+### Add NPM hook
+To allow `npm test` to run your unit tests, simply call `roca` in your `package.json`'s "test" script:
+
+```jsonc
+{
+    // ...
+    scripts: {
+        "test": "roca"
+    }
+}
+```
+
+### Directory Layout
+Roca tests must be in a *sibling* of your existing `source/` and `components/` tree, like this:
+
+
+```
+.
+├── components
+│  ├── FooComponent.brs
+│  └── FooComponent.xml
+├── manifest
+├── source
+│  ├── main.brs
+│  └── util.brs
+└── tests
+   ├── FooComponent.test.brs
+   └── util.test.brs
+```
+
+Additionally, `roca` should be called from the same directory as your `manifest` file.  If you keep your entire source tree in a subdirectory, you'll want to `cd` into that before executing `roca`.
+
+### Modify Channel Entry Point (if necessary)
+Roca only supports a main channel entry point of `runUserInterface`, which helpfully avoids naming conflicts with the `main` functions used in `roca` and the individual test files.
+
+Luckily `runUserInterface` is supported for all the same use-cases as `main`: https://developer.roku.com/docs/developer-program/getting-started/architecture/dev-environment.md#sub-runuserinterface
+
+### Adding a Test
+Just like [Mocha](https://mochajs.org/) tests are written in JavaScript, roca tests are written in BrightScript.  Roca will find and execute all files in the `./tests/` directory that match `*.test.brs`.  The smallest possible unit test must meet the following requirements:
```

Comment:
Tiniest little nit of all time: consistent capitalization (mocha and roca, or Mocha and Roca)

Reply:
one's the start of a sentence 😉 but yeah, we should probably formalize that!